iron-icon.html

```diff
@@ -89,6 +100,7 @@
 
       width: var(--iron-icon-width, 24px);
       height: var(--iron-icon-height, 24px);
+      color: var(--iron-icon-color, inherit);
```

Comment:
I thought the color was being given by the `fill: currentcolor;` style right above. Is that still needed? Do they work together?